app/views/users/_user.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="user-list-container">
+    <div class="user-image-container">
+         <image class="user-photo" src="<%= user.photo %>"></image>
+    </div>
+    <div class="user-detail-container">
+        <h3 class="user-name"><%= user.name %></h3>
+        <span class="num-posts">Number of posts: <%= user.posts_counter %></span>
+    </div>
+</div>
```

Comment:
- [ ] Clicking on a user should take me to their corresponding show page. Kindly make this user name or div class a clickable link that fulfills the above specification.